src/theme/sizes.js

```diff
@@ -0,0 +1,62 @@
+import { Dimensions } from 'react-native';
+
+const { height, width } = Dimensions.get('screen');
+
+const SIZES = {
+  BASE: 16,
+  FONT: 16,
+  ICON: 16,
+  OPACITY: 0.8,
+  BORDER_RADIUS: 6,
+  BORDER_WIDTH: 0.8,
+
+  // Button styles
+  BUTTON_WIDTH: 16 * 9,
+  BUTTON_HEIGHT: 16 * 2.75,
+  BUTTON_SHADOW_RADIUS: 10,
+
+  // Block styles
+  BLOCK_SHADOW_OPACITY: 0.15,
+  BLOCK_SHADOW_RADIUS: 8,
+  CARD_BORDER_RADIUS: 16 * 0.4,
+  CARD_BORDER_WIDTH: 16 * 0.05,
+  ELEVATION: 1,
+
+  // Card styles
+  CARD_WIDTH: width - (16 * 2),
+  CARD_MARGIN_VERTICAL: 16 * 0.875,
+  CARD_FOOTER_HORIZONTAL: 16 * 0.75,
+  CARD_FOOTER_VERTICAL: 16 * 0.75,
+  CARD_AVATAR_WIDTH: 16 * 2.5,
+  CARD_AVATAR_HEIGHT: 16 * 2.5,
+  CARD_AVATAR_RADIUS: 16 * 1.25,
+  CARD_IMAGE_HEIGHT: 16 * 12.5,
+  CARD_ROUND: 16 * 0.1875,
+  CARD_ROUNDED: 16 * 0.5,
+
+  // Input styles
+  INPUT_BORDER_RADIUS: 16 * 0.5,
+  INPUT_BORDER_WIDTH: 16 * 0.05,
+  INPUT_HEIGHT: 16 * 2.75,
+  INPUT_HORIZONTAL: 16,
+  INPUT_TEXT: 16 * 0.875,
+  INPUT_LABEL_TEXT: 16 * 0.9,
+  INPUT_LABEL_BOTTOM: 16 / 4,
+  INPUT_HELP_TEXT: 16 * 0.8,
+  INPUT_ROUNDED: 16 * 1.7,
+
+  // NavBar styles
+  NAVBAR_HEIGHT: 16 * 4.125,
+  NAVBAR_VERTICAL: 16,
+  NAVBAR_TITLE_FLEX: 2,
+  NAVBAR_TITLE_HEIGHT: height * 0.07,
+  NAVBAR_TITLE_TEXT: 16 * 0.875,
+  NAVBAR_LEFT_FLEX: 0.5,
```

Comment:
Combine NAVBAR_LEFT_* & NAVBAR_RIGHT_* as they are 'always' the same?

Reply:
Are the same at this point, for better customisation anyone can change the value using directly this constants

Reply:
@paullaros @palingheorghe please check the implementation of the theme & components.

Reply:
@paullaros @palingheorghe idea: add theme as separate npm package, e.g.: galio-theme
What do you think?

Reply:
@paullaros @palingheorghe 
- **galio-framework** is at version **1.1.0**
- **expo** version **31.0.4**
- **react** version **16.5.0**

**examples** branch will need to upgrade to version newest expo version